src/utils/enums/types.ts

```diff
@@ -0,0 +1,18 @@
+export enum ShapeType {
+    circle = "circle",
+    edge = "edge",
+    triangle = "triangle",
+    polygon = "polygon",
+    star = "star",
+    image = "image",
+    square = "square",
+    heart = "heart",
+    char = "char",
+    character = "character",
+    line = "line",
+}
+
+export enum ProcessBubbleType {
+    size = "size",
+    opacity = "opacity",
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5012291)